tests/validation_suite/worker_tests/test_worker_update_api.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import logging
+import json
+
+import worker.worker_details as worker
+from automation_framework.utilities.process_api_request import \
+                                    process_api_request
+from automation_framework.utilities.request_args import TestStep
+
+logger = logging.getLogger(__name__)
+
+
+def test_worker_update(setup_config):
+    """ Testing worker update request with all valid parameter values. """
+
+    # retrieve values from conftest session fixture
+    worker_obj = setup_config[0]
```

Comment:
Each test case will have this code, please try to avoid duplicate code in all test case. Move this code to function and call that function.